app/assets/stylesheets/custom.css.scss

```diff
@@ -135,44 +141,88 @@ h1 {
   background-color: darkgrey;
 }
 
-table td {
-  border: 2px solid black;
-  padding: 10px;
-  align: center;
-}
-
-.admin_orders_list {
-  width: 65%;
-  padding: 5px;
-  border: 1px solid;
-  height: 300px;
-  overflow: auto;
-}
-
+// table td {
+//   font-family: monospace;
+//   border: 2px solid black;
+//   padding: 10px;
+//   align: center;
+// }
+//
+// .admin_orders_list {
+//   font-family: monospace;
+//   width: 65%;
+//   padding: 5px;
+//   border: 1px solid;
+//   height: 300px;
+//   overflow: auto;
+// }
+//
 .user_orders_list {
+  font-family: monospace;
   width: 65%;
   padding: 5px;
   border: 1px solid;
   height: 300px;
   overflow: auto;
 }
-.center_div {
-  padding: 3px;
-  margin-right: 10px;
+// .center_div {
+//   font-family: monospace;
+//   padding: 3px;
+//   margin-right: 10px;
+// }
+//
+// .dropdown-menu li:hover .sub-menu {
+//   font-family: monospace;
+//     visibility: visible;
+// }
+//
+// .dropdown:hover .dropdown-menu {
+//   font-family: monospace;
+//     display: block;
+// }
+//
+// .account{
+//     float:left
+// }
+//
+.admin_orders_list h3 {
+  font-family: monospace;
+  text-align: center;
+}
+//
+// .dashboard {
+//   float: left;
+//   margin-left: 5%;
+// }
+//
+// .index {
+//   float: left;
+//   list-style: none;
+//   margin-left: 10px;
+// }
+//
+.container h1 {
+  text-align: center;
 }
 
-.dropdown-menu li:hover .sub-menu {
-    visibility: visible;
+.container {
+  font-family: monospace;
 }
 
-.dropdown:hover .dropdown-menu {
-    display: block;
+.container .index{
```

Comment:
Merge rule `.container .index` with rule on line 208<br>Opening curly brace `{` should be preceded by one space